kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -14,12 +14,10 @@
       <mat-svg id="edit-icon" class="name-edit" category="image" name="edit" aria-hidden="true"/>
     </div>
 
-    <hr>
-
     <div class="header">
-      <h1>
+      <h2 class="table-title">
         {{$tr('tableTitle')}}
-      </h1>
+      </h2>
```

Comment:
Was the point of this change to adjust the font size or the heading level?

If it was a stylistic font change, I believe we're supposed to leave this as an `h1`